frontend/src/App.tsx

```diff
@@ -96,7 +131,10 @@ const App: React.FC = () => {
   return (
     <div className="App">
       <header className="App-header">
-        <div>WebRTC Hello World</div>
+        <div>WebRTC Voice Calls - using asynchronous Bridge</div>
```

Comment:
Capitalize U and A... It looks more consistent

Reply:
Or make B lowercase

Reply:
b lowercase.